plugin-api/MetaflowPluginAPI.js

```diff
@@ -73,73 +73,93 @@ const Metaflow = {
+   * @param {string[]} paths
+   * @param {(event: { path: string, data: * }) => void} fn
    */
   subscribe(paths, fn) {
     Listeners.push(fn);
-    window.parent.postMessage({ name: window.name, type: 'PluginSubscribeToData', paths: paths }, '*')
+    window.parent.postMessage({ name: window.name, type: 'PluginSubscribeToData', paths: paths }, '*');
   },
   /**
-   * Subsribe to events 
+   * Subsribe to events
    * @param {string[]} events List of event name to subscribe to
    * @param {(event: { type: string, data: * }) => void} fn Callback to trigger in case of event
    */
   on(events, fn) {
     EventListeners.push(fn);
-    window.parent.postMessage({ name: window.name, type: 'PluginSubscribeToEvent', events: events }, '*')
+    window.parent.postMessage({ name: window.name, type: 'PluginSubscribeToEvent', events: events }, '*');
   },
   /**
    * Call event with any name and payload. Other plugins or systems in app might subscribe to these events.
    * @param {string} event
-   * @param {*} data 
+   * @param {*} data
    */
   call(event, data) {
-    window.parent.postMessage({ name: window.name, type: 'PluginCallEvent', event: event, data: data }, '*')
+    window.parent.postMessage({ name: window.name, type: 'PluginCallEvent', event: event, data: data }, '*');
   },
   /**
    * Send notification on main application
-   * @param {string | {type: string, message: string}} message 
+   * @param {string | {type: string, message: string}} message
    */
   sendNotification(message) {
-    window.parent.postMessage({ name: window.name, type: 'PluginCallEvent', event: 'SEND_NOTIFICATION', data: message }, '*')
+    window.parent.postMessage(
+      { name: window.name, type: 'PluginCallEvent', event: 'SEND_NOTIFICATION', data: message },
+      '*',
+    );
   },
   /**
    * Update visibility of plugin. It will remain in DOM either way.
-   * @param {boolean} visible 
+   * @param {boolean} visible
    */
   setVisibility(visible) {
-    window.parent.postMessage({
-      name: window.name, type: 'PluginCallEvent', event: 'UPDATE_PLUGIN', data: {
-        slot: PluginInfo.slot,
-        name: PluginInfo.manifest.name,
-        visible: visible
-      }
-    }, '*')
+    window.parent.postMessage(
+      {
+        name: window.name,
+        type: 'PluginCallEvent',
+        event: 'UPDATE_PLUGIN',
+        data: {
+          slot: PluginInfo.slot,
+          name: PluginInfo.manifest.name,
+          visible: visible,
+        },
+      },
+      '*',
+    );
   },
   //
   // Request to be removed?
   //
   remove(fn) {
-    window.parent.postMessage({ name: window.name, type: 'PluginRemoveRequest' }, '*')
+    window.parent.postMessage({ name: window.name, type: 'PluginRemoveRequest' }, '*');
   },
-}
 
-if (typeof exports !== "undefined") {
-  exports.Metaflow = Metaflow
+  subscribeToMetadata(fn) {
+    this.subscribe(['metadata'], fn);
+  },
+
+  subscribeToRunMetadata(fn) {
+    this.subscribe(['run-metadata'], fn);
```

Comment:
I don't see `run-metadata` in the documentation anywhere. I think we should have it listed with `metadata` somewhere so plugin writers don't have to consult to code to find it.